stdlib/tkinter/__init__.pyi

```diff
@@ -3206,12 +3206,19 @@ class OptionMenu(Menubutton):
     # configure, config, cget are inherited from Menubutton
     # destroy and __getitem__ are overridden, signature does not change
 
-class _Image(Protocol):
-    tk: _tkinter.TkappType
-    def height(self) -> int: ...
-    def width(self) -> int: ...
+# Marker to indicate that it is a valid bitmap/photo image. PIL implements compatible versions
+# which don't share a class hierachy. The actual API is a __str__() which returns a valid name,
+# not something that type checkers can detect.
+@type_check_only
+class _Image: ...
```

Comment:
I'm not sure if defining a "fake" base class has some surprising consequences that I'm not aware of. Stubbing something differently than what the runtime does is often a bad idea, but here we need to do it anyway. But this seems like a good idea and I think we should try it.

That said, I don't think a protocol with `width(self) -> int` and `height(self) -> int` would ever match the wrong kind of object in practice. Most libraries would define `width` and `height` as properties or attributes rather than as methods that take no arguments.